bundles/org.openhab.binding.omnikinverter/README.md

```diff
@@ -21,11 +21,20 @@ No autodiscovery available
 
 ## Channels
 
-| Channel Type Id   | Item Type     | Description                          |
-| :---------------- | :----------   | :------------                        |
-| power             | Number:Power  | The instantaneous power generation   |
-| energyToday       | Number:Energy | The amount of energy generated today |
-| energyTotal       | Number:Energy | The total amount of energy generated |
+| Channel Type Id | Item Type      | Description                                                                          |
+| :---------------| :--------------| :------------------------------------------------------------------------------------|
+| power           | Number:Power   | The instantaneous power generation for feed 1 to the grid in Watt (same as powerAC1) |
+| powerAC1        | Number:Power   | The instantaneous power generation for feed 1 to the grid in Watt                    |
+| powerAC2        | Number:Power   | The instantaneous power generation for feed 2 to the grid in Watt                    |
+| powerAC3        | Number:Power   | The instantaneous power generation for feed 3 to the grid in Watt                    |
+| currentPV1      | Number:Current | The current generation for input string 1 in ampere                                  |
+| currentPV2      | Number:Current | The current generation for input string 2 in ampere                                  |
+| currentPV3      | Number:Current | The current generation for input string 3 in ampere                                  |
+| voltagePV1      | Number:Voltage | The voltage on input string 1 in volt                                                |
+| voltagePV2      | Number:Voltage | The voltage on input string 2 in volt                                                |
+| voltagePV3      | Number:Voltage | The voltage on input string 3 in volt                                                |
+| energyToday     | Number:Energy  | The amount of energy generated today in kWh                                          |
+| energyTotal     | Number:Energy  | The total amount of energy generated in kWh                                          |
```

Comment:
The user will have the choice of the unit. So maybe add for example "in ampere by default".